README.md

```diff
@@ -3,3 +3,6 @@
 # Backend [![Build Status](https://travis-ci.org/mariogintili/backend.svg?branch=master)](https://travis-ci.org/mariogintili/backend) [![Code Climate](https://codeclimate.com/github/mariogintili/backend/badges/gpa.svg)](https://codeclimate.com/github/mariogintili/backend)
 
 A Rails starter app, [JSON-API](http://jsonapi.org/) compliant + [OAUTH 2](http://oauth.net/2/) based authentication :monkey_face:  Always riding the latest edge :light_rail:
+
+##Caveats
+Due to a Rails bug, in order to correctly serve it for the [client counterpart](https://github.com/mariogintili/ember-oauth-client) Ember app, run the server like so: `rails s --binding 127.0.0.1`.
```

Comment:
can you please change `rails s` to `bin/rails`? This is to follow the new guidelines rails requires :) since it uses the CLI command corresponding to the rails version of your local project.